bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -69,6 +63,14 @@ class PlotCanvasView extends Renderer.View
     super(options)
     @pause()
 
+    # TODO (bev) this sucks a bit
+    @visuals = {}
+
+    for spec in @model.plot.mixins
```

Comment:
If I'm not mistaken this is duplicating code from Renderer.

Can I suggest that we just break out a method and reuse it in both places.